### PR TITLE
Fixed all html lint errors

### PR DESCRIPTION
**Pull request by adarsh-ideal**
Fixed all html lint errors caused by this issue: https://github.com/fossasia/apps.loklak.org/issues/53.
Fixes a part of this issue: https://github.com/fossasia/apps.loklak.org/issues/82.
@mariobehling @daminisatya Please review this.
I got a green build when disabled css lint, Build: https://travis-ci.org/adarsh-ideal/apps.loklak.org/builds/193811266
@rhemon is solving css lint errors and this issue will be soon resolved.

Then I would like to add javascript lint as well.

### CODE REVIEW

File: LQL/index.html [hunk 1]
```diff
@@ -36,37 +31,44 @@
...
-
               <!-- Form Name -->
-              
-
               <!-- Text input-->
               <div class="form-group">
-                <label class="col-md-4 control-label" for="query">Query Text</label>  
+                <label class="col-md-4 control-label" for="query">Query Text</label>
                 <div class="col-md-8">
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline LQL/index.html]: Fix this conflict marker, please.

File: NMEA/index.html [hunk 2]
```diff
@@ -1,140 +1,180 @@
...
-		height: 1000px;
-	}
-	</style>
-	<script src="../../js/angular.min.js"></script>
+<meta charset="UTF-8">
+<title>NMEA Visualizer</title>
+<link rel="stylesheet" href="leaflet.css">
+<script src="leaflet.js"></script>
+<meta charset="utf-8">
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Fix this conflict marker

File: NMEA/index.html [hunk 3]
```diff
@@ -1,140 +1,180 @@
...
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="Content-Language" content="en, english">
+<meta name="author" content="@sudheesh001"> <!-- CHANGE THIS -->
+<link rel="icon" type="image/png" href="../../artwork/favicon.png">
+<link href="../../css/bootstrap.min.css" rel="stylesheet">
+<script src="../../js/jquery.js"></script>
+<script src="../../js/bootstrap.min.js"></script>
+<style>
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Fix this conflict marker

File: NMEA/index.html [hunk 4]
```diff
@@ -1,140 +1,180 @@
...
-			<div id="map"></div>
-		</div>
+    <div class="row">
+<div id="map"></div>
+</div>
   <!-- Modal -->
   <div class="modal fade" id="streamUrl" role="dialog">
     <div class="modal-dialog">
-    
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Fix this conflict marker

File: NMEA/index.html [hunk 5]
```diff
@@ -1,140 +1,180 @@
...
+        <div class="col-md-10">
+>>>>>>> add0f2f... Fixed html lint errors
+<input id="url" name="url" type="url" placeholder="Enter the data stream URL here" class="form-control input-md">
+</div>
+<div class="col-md-2">
+<input type="submit" value="Submit" onclick="getTracking();" class="btn btn-info">
+</div>
+</div>
+</div>
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Fix this conflict marker

File: NMEA/index.html [hunk 6]
```diff
@@ -1,140 +1,180 @@
...
+  </div>
+</div>
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Fix this conflict marker

File: NMEA/index.html [hunk 7]
```diff
@@ -1,140 +1,180 @@
...
-		var url = document.getElementById('url').value;
+</div>
+<script>
+$(document).ready(function(){
+    $("#streamUrl").modal();
+});
+
+function getTracking() {
+var url = document.getElementById("url").value;
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Fix this conflict marker

File: NMEA/index.html [hunk 8]
```diff
@@ -1,140 +1,180 @@
...
-	}
+>>>>>>> add0f2f... Fixed html lint errors
+var urlComplete = baseUrl+"/api/nmea.txt?stream="+url;
+var centerlat = 52;
+var centerlon =  0;
+// set default zoom level
+var zoomLevel = 2;
+$.getJSON(urlComplete, function (data) {
+var GPSObjects = [];
+for (var key in data) {
```
Comment (yukiisbored) [inline NMEA/index.html]: Your patch makes the code unreadable, fix this.

File: NMEA/index.html [hunk 9]
```diff
@@ -1,140 +1,180 @@
...
 var zoomLevel = 5;
 
 // initialize map
-var map = L.map('map').setView([centerlat, centerlon], zoomLevel);
+var map = L.map("map").setView([centerlat, centerlon], zoomLevel);
 
 // Set up the OSM layer
 var background = L.tileLayer(
-    'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', 
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Fix this conflict marker

File: NMEA/index.html [hunk 10]
```diff
@@ -1,140 +1,180 @@
...
+    layer.bindPopup("<table style="width:180px"><tbody><tr><td><div><b>name:</b></div></td><td><div>" + feature.properties.popup + "</div></td></tr><tr class><td><div><b>time:</b></div></td><td><div>" + feature.properties.time + "</div></td></tr></tbody></table>");
 }
 
 function connectTheDots(data){
     var c = [];
     for(i in data._layers) {
-        var x = data._layers[i]._latlng.lat;
-        var y = data._layers[i]._latlng.lng;
-        c.push([x, y]);
+  var x = data._layers[i]._latlng.lat;
```
Comment (yukiisbored) [inline NMEA/index.html]: Why is this indented below the level of the `for` loop?

File: NMEA/index.html [hunk 11]
```diff
@@ -151,26 +191,54 @@ <h4 class="modal-title">Enter the stream URL</h4>
...
     var y = init_pt[0];
     var c = [
-        [x, y]
+  [x, y]
     ];
     var angle = init_angle;
     var dist = init_dist;
 
     for (var i = 0; i < steps; ++i) {
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline NMEA/index.html]: Solve this conflict marker

File: WebScraper/index.html [hunk 12]
```diff
@@ -18,46 +18,52 @@
                <div class="input-group" id="searchBar" style="width:600px;">
                   <input type="text" id="url" class="form-control" ng-model="url" placeholder="Enter website address">
                   <span class="input-group-btn">
-                  <button href='#' class="btn btn-default" type="button" id="button" ng-click="Scrape()">Convert</button>   
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline WebScraper/index.html]: Solve this conflict marker

File: WebScraper/index.html [hunk 13]
```diff
@@ -18,46 +18,52 @@
...
             <br>
             <br>
          </div>
       </center>
       <div ng-if="data" style="padding-left:28%;">
-        <json-formatter json="data" open="1"></json-formatter>
+        <json-formatter json="data" open></json-formatter>
       </div>
-	  
+<<<<<<< HEAD
```
Comment (yukiisbored) [inline WebScraper/index.html]: Solve this conflict marker

**Comment by yukiisbored** [comment]
Please solve *all* of the conflict markers, there's a ton of them and I'm not in the mood to pin all of them because It's a simple `git grep '<<<<<<< HEAD'` to find all of them. Also please configure your linter to make the patch as small as possible (unless there's an actual inconsistency in code styles).

File: barchart/index.html [hunk 14]
```diff
@@ -1,194 +1,175 @@
...
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+  <div class="container-fluid">
+  <div class="navbar-header">
+  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+  <span class="sr-only">Toggle navigation</span>
+  <span class="icon-bar"></span>
+  <span class="icon-bar"></span>
+  <span class="icon-bar"></span>
+  </button>
+<<<<<<< HEAD
```
Comment (rhemon) [inline barchart/index.html]: resolve the conflicts here

File: barchart/index.html [hunk 15]
```diff
@@ -1,194 +1,175 @@
...
+  <div id="navbar" class="navbar-collapse collapse">
+<ul class="nav navbar-nav navbar-right">
+</ul>
+  </div>
+  </div>
+  </nav>
+  <!-- Fancy Form -->
+  <div ng-controller="MainCtrl as main" ng-cloak="">
+  <div layout="column">
+<<<<<<< HEAD
```
Comment (rhemon) [inline barchart/index.html]: here as well

File: barchart/index.html [hunk 16]
```diff
@@ -1,194 +1,175 @@
...
+  <form ng-submit="poll(query.type, query.term, query.interval, query.dateFrom, query.dateTo, query.maxHashtags, query.count)"
+=======
+  <form ng-submit="poll(query.type, query.term, query.interval, query.dateFrom, query.dateTo, query.maxHashtags, query.count)" 
+>>>>>>> add0f2f... Fixed html lint errors
+  class="layout-margin layout-fill" name="queryForm">
+  <!-- form name for ng-messages -->
+  <md-content md-theme="docs-dark" layout-gt-sm="row" layout-xs="column" layout-padding layout="center center">
+  <div flex="10"></div>
+<<<<<<< HEAD
```
Comment (rhemon) [inline barchart/index.html]: conflict

File: index.html [hunk 17]
```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html lang="en" ng-app="appListApp">
   <head>
-        <meta charset="utf-8">
+    <meta charset="utf-8">
     <meta name="robots" content="index,follow">
+<<<<<<< HEAD
```
Comment (rhemon) [inline index.html]: conflict

**Comment by rhemon** [comment]
@adarsh-ideal there are merge conflicts, some files are not  mentioned here, so please go through the code and fix the conflicts. also it would be good if you don't remove the indentations in your code change.
I am nearly done with my part of code fix ([code change](https://github.com/rhemon/apps.loklak.org/pull/7/files) and [travis log](https://travis-ci.org/rhemon/apps.loklak.org), once you finish this part I will cherry-pick this, which will probably give lots of merge conflicts 😅  and then I will also have to rebase which will bring more problems. :P 
but we should get this done quickly

**Comment by rhemon** [comment]
I have taken all changes to #108 , this can be closed

**Pull request closed without merging**